requirements.txt

```diff
@@ -1 +1 @@
-requests==2.25.0
+beautifulsoup4==4.9.1
```

Comment:
This one is supplied by the platform as well

Reply:
Sure, Removed